multiversx_sdk_cli/cli_contracts.py

```diff
@@ -433,6 +457,18 @@ def query(args: Any):
     utils.dump_out_json(result)
 
 
+def _get_contract_arguments(args: Any) -> Tuple[List[Any], bool]:
+    json_args = json.loads(Path(args.arguments_file).expanduser().read_text()) if args.arguments_file else None
+
+    if json_args and args.arguments:
+        raise Exception("Both '--arguments' and '--arguments-file' provided.")
```

Comment:
Can be rephrase as suggestion for correction e.g. "provide either ... or ...".

Reply:
rephrased